HACKING.md

```diff
@@ -324,61 +288,60 @@ File Attributes
   Tag_CPU_arch: v7
   Tag_FP_arch: VFPv3-D16
 ```
-### Contributing
+
+## Contributing
 
 Contributions are always welcome!
 
 Please make sure that you sign the Canonical contributor agreement [here](
 http://www.ubuntu.com/legal/contributors).
 
-Complete requirements can be found in CONTRIBUTING.md.
+Complete requirements can be found in [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-Contributions are submitted through a Pull Request created from a fork of the
-`snapd` repository (under your GitHub account). 
+### Guidelines for contributors
 
-Start by creating a fork of the `snapd` repository on GitHub.
+There are few guidelines that contributors should follow:
 
-Add your fork as an additional remote to an already cloned `snapd` main
-repository. Replace `<user>` with your GitHub account username.
+- Commit messages should be well structured,  
```

Comment:
@MiguelPires, I am not sure I agree. Once again it's not a rule here, but just a recommendation. It's very simple, please look at it.

Reply:
> @MiguelPires, I am not sure I agree. Once again it's not a rule here, but just a recommendation. It's very simple, please look at it.

You don't agree with changing ',' to '.' ? Also, what does "it's very simple, please look at it" mean? What did I not look at?

Reply:
@MiguelPires, I assumed that you suggest to put here "." since you suggest to omit the reference to the the [Conventional Commits](https://www.conventionalcommits.org/). Therefore, I asked you to look at it, I mean at the specification.

Reply:
No, it's just changing the comma for the period since every other list item uses a period

Reply:
As I expected, I simply didn't understand you right. :flushed: 

Buy the way, I would love to hear you opinion on this specification?:smiley: